core/src/stores/postgres.rs

```diff
@@ -2390,12 +2390,19 @@ impl Store for PostgresStore {
             )
             .await?;
 
-        for (row_id, content) in contents {
-            let row_created = utils::now();
-            let row_data = content.to_string();
+        for row in rows {
```

Comment:
while we're here, can we iter collect that  ?

Reply:
won't change much as it would be an iter.forEach()